app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1><%=@task.name%></h1>
+
+<h2><%=@task.description%></h2>
+
+<h3>Completed: <%=@task.completed_at%></h3>
```

Comment:
This isn't a good user experience! you need to link off of this page either here or from the `application.html.erb` otherwise the users can only navigate back to the main page with the url.